packages/lib/gpcircuits/src/proto-pod-gpc.ts

```diff
@@ -449,6 +469,8 @@ export class ProtoPODGPC {
     return undefined;
   }
 
+  // TODO(POD-P2): Replace `ProtoPODGPCCircuitParams` with a different type
```

Comment:
I think I deleted this comment in another PR.  Did you add it back on purpose, or was it an auto rebase?
I think we've got a workable solution for param requirements for the current features, unless you think there's more to do.

Reply:
Auto rebase I guess!